vw/src/main/scala/com/eharmony/spotz/objective/vw/VwFunctions.scala

```diff
@@ -91,6 +91,13 @@ trait VwFunctions {
     }
   }
 
+  def getContextualBanditActions(vwParamMap: Map[String, _]): Option[Int] = {
+    vwParamMap.get("cb") match {
```

Comment:
I would bet this isn't general enough.  I would try `--cb_adf`, `--cb_explore`, and `--cb_explore_adf`.